KS-09-03-servings-count

DenisGubernatorov opened the pull request:
1) Расчет необходимого количества ингредиентов на количество порций омлета, указанное пользователем

DenisGubernatorov committed:
KS-09-03-servings-count
1) Расчет необходимого количества ингредиентов на количество порций омлета, указанное пользователем

ievetrovdev commented on src/main/kotlin/lesson9/lesson9_task3.kt:
```diff
@@ -1,2 +1,19 @@
+
+    print("Введите количество порций: ")
+    val servingCount = readln().toInt()
+    servings.replaceAll { it * servingCount }
```
Используй функцию map для этих целей) и в каждую итерацию будем перемножать элемент коллекции

DenisGubernatorov replied:
Что бы не зависать на задаче, перевел на использование map.

DenisGubernatorov committed:
KS-09-03-servings-count
1) Перевод подсчета необходимого количества ингредиентов на использование функции map
2) Рефакторинг имен, исходя из новой логики
